doc/source/groups_of_atoms.rst

```diff
@@ -0,0 +1,117 @@
+.. -*- coding: utf-8 -*-
+.. _groups-of-atoms:
+
+Groups of atoms
+===============
+
+MDAnalysis has a hierarchy of :class:`~MDAnalysis.core.groups.Atom` containers that are used throughout the code.
+
+.. image:: images/classes.png
+
+First and foremost is the :class:`~MDAnalysis.core.groups.AtomGroup`. An :class:`~MDAnalysis.core.groups.AtomGroup` is the primary :class:`~MDAnalysis.core.groups.Atom` container; virtually everything can be accessed through it, as detailed `above <https://www.mdanalysis.org/UserGuide/data_structures.html#atomgroup>`_. This includes chemically meaningful groups of :class:`~MDAnalysis.core.groups.Atom`\ s such as a :code:`Residue` or a :code:`Segment`. 
+
+---------------------
+Residues and Segments
+---------------------
+
+A :class:`~MDAnalysis.core.groups.Residue` is composed of :class:`~MDAnalysis.core.groups.Atom`\ s, and a :class:`~MDAnalysis.core.groups.Segment` is composed of :class:`~MDAnalysis.core.groups.Residue`\ s.
+
+These are considered atom :ref:`topology attributes <topology-attributes>`.
```

Comment:
This sentence might be true but is confusing unless you are someone who has had nightmares over topology attributes and what they are... I would move it further down, here it seems to distract from the "container" aspect.

Reply:
Or use a footnote.